NEWS.rst

```diff
@@ -1,6 +1,16 @@
 History
 =======
 
+0.14 (2019-10-02)
+-----------------
+* Make L2 product by applying self-calibration corrections (#253 - #256)
+* Speed up uvw calculations (#252, #262)
+* Produce documentation on readthedocs.org (#244, #245, #247, #250, #261)
+* Clean up mvftoms and fix REST_FREQUENCY in SOURCE sub-table (#258)
+* Support katstore64 API (#265)
+* Improve chunk store: detect short reads and speed up lost data (#259, #260)
```

Comment:
maybe "... speed up handling of lost data."

Reply:
Good idea.